### PR TITLE
Build a 199 tag

### PR DESCRIPTION
**Pull request by eiriksm**
## What is the objective of this PR

Useful in case latest stops working 

## Ny Media Pull Request Checklist (NMPRC)

- [ ] Did you create the pull request to be merged into the [correct target branch](https://github.com/nymedia/.github/blob/master/pull_request_template_explained.md#branching)?
- [ ] Is it possible to deploy automatically? If it requires [manual deployment](https://github.com/nymedia/.github/blob/master/pull_request_template_explained.md#manual-deployment), did you include the steps needed to deploy?
- [ ] Did you [manually test](https://github.com/nymedia/.github/blob/master/pull_request_template_explained.md#manual-testing) the change?
- [ ] Is the change to a module that only affects this project? If the change is to a module shared across projects, did you manually test the change in [another project's repository](https://github.com/nymedia/.github/blob/master/pull_request_template_explained.md#manual-testing)?
- [ ] Did you make sure documentation have been updated, if needed?
- [ ] Did you include all necessary translations, if any?
- [ ] Did you make sure to update sanitization routines if fields for storing personal information are added?

If you have suggestions for how this Pull Request Template can be better, please create an issue (or a pull request) on [https://github.com/nymedia/.github](https://github.com/nymedia/.github)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
          # Ref: https://github.com/Zenika/alpine-chrome
         chromium_version:
           - 'latest'
-          - '89'
+          - '119'
```
Comment (esolitos) [inline .github/workflows/build.yml:25]: Why replacing 'tho? :)

Comment (eiriksm) [reply .github/workflows/build.yml:25]: Simply because I don't see why we need to periodically build an almost 3 year old browser 🤓😅

Comment (esolitos) [reply .github/workflows/build.yml:25]: That is indeed not an unreasonable argument. 😄 

What gets 'tho updated is the underlying alpine, it's not necessarily critical, but imho still not bad. :)

Comment (eiriksm) [reply .github/workflows/build.yml:26]: no strong opinions. i added it back

**Comment by esolitos** [comment]
✔️ 
https://github.com/nymedia/chrome-headless/pkgs/container/chrome-headless/163943631?tag=119

File: .github/workflows/build.yml (hunk 1 above)
Comment (esolitos) [reply .github/workflows/build.yml:26]: I released that there's another thing which changes other than "the base alpine": the `chromium-chromedriver` is Independent from chrome itself: that one is updated on each run.

**Pull request merged**